setup.py

```diff
@@ -15,8 +16,7 @@
                   Please get the latest version of Cython and try again!')
 
 try:
-    if (os.environ['TOX'] == 'true' and
```

Comment:
Maybe it would be cleaner to use some obvious variable here, like if os.environ.get('CYTHON_TRACE'), and define this variable in tox environments where it's needed?

Reply:
I just found out that we can get the platform information with **platform.python_implementation()**, what do you think @lopuhin ?

Reply:
That's nice, but I think we don't want to enable tracing on CPython either, unless it's needed for coverage report?

Reply:
Oh yes that is true... I just changed the code so it uses env var for enabling cython linetrace and **platform.python_implementation()** for not enabling cython linetrace in pypy. What do you think?

Reply:
Yes, looks good! Note that instead of 
```
try:
    if os.environ['CYTHON_TRACE']:
except KeyError as e:
```
you can use ``if os.environ.get('CYTHON_TRACE'):``

Reply:
haha thank you so much for that! I was struggling with this if statement 😆